.env.test

```diff
@@ -0,0 +1,6 @@
+MYSQL_DATABASE=employees
+MYSQL_HOST=db
+MYSQL_PORT=3306
```

Comment:
This file should be ignored by the .gitignore. Can you add the dotenv file patterns to the .gitignore file and remove this file from the tracked files ?
No need to squash the commits, these secrets are for test purposes.

Reply:
I think we better keep the .env.test in the repo as it facilitates the tests launch.
Otherwise, One should have to know what to provide as HOST_NAME, PORT...
In the future, we may want to test other services, and it will be painful to know what environment they require before being able to launch the integration test suit.
Security is here to protect data we care about, no random test data with container lifetime.

Reply:
You're right about the user being able to perform the tests without having to fill in anything but there is still an issue by having a .env file and a .env.test file that don't include the same variables.
Let's keep it simple and just fill in the variables directly in the docker-compose.yml file then (and we could actually rename it into docker-compose.test.yml)

https://github.com/motdotla/dotenv#should-i-have-multiple-env-files